README.md

```diff
@@ -248,7 +248,13 @@ release "orchestrator" uninstalled
 ```
 
 Note that the CRDs created during the installation process will remain in the cluster.
-To clean the rest of the resources, run:
+
+Before cleaning the rest of the resources, if you have workflows installed, you will need to remove them from the cluster first:
+```console
+helm delete <workflow>
```

Comment:
@gabriel-farache 
this doesn't cover the use case of creating workflows without helm
for example - using the backstage Create from template, which is the recommended way to do so

Reply:
@batzionb You are right, I was continuing to investigate the issue and it appears I can only reproduce the removal issue if the M2K workflow (and its Knative resources) were installed

So the issue seems to be related to workflow having Knative resources installed as well

I asked Gary to confirm he also had the M2K installed when he encountered the issue. In the meantime, I updated the PR to reflect my discovery and I put it in draft while waiting confirmation from Gary

Reply:
and AFAIK, there is no way to install Knative resources using the bootstrap project/templates right now so I do no see the issue happening in that case